edx_proctoring/static/proctoring/templates/course_allowances.underscore

```diff
@@ -0,0 +1,58 @@
+<span class="tip"> <%- gettext("Add Allowance") %>
```

Comment:
I find it a bit strange to have "Add Allowance" twice, once in the bar and again in the button. 
Perhaps here: "Add Allowance" -> "Allowances"